CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * **Removed**
    * REMOVED: Remove Node bindings. [#2502](https://github.com/valhalla/valhalla/pull/2502)
    * REMOVED: appveyor builds. [#2544](https://github.com/valhalla/valhalla/issues/2544)
+   * REMOVED: Removed x86 CI builds. [#2792](https://github.com/valhalla/valhalla/issues/2792)
```

Comment:
ugh.. not `issues`, `pulls`.. makes me want to do this: https://github.com/valhalla/valhalla/pull/2763#issuecomment-756804568.. all these wasted CI hours for dumb commits like this

Reply:
looks like the one above is the same too, should be: https://github.com/valhalla/valhalla/pull/2550